data/ships.txt

```diff
@@ -1480,6 +1480,46 @@ ship "Gunboat"
 
 
 
+ship "Harvester"
+	sprite "ship/mining drone"
+	attributes
+		category "Drone"
+		"cost" 100000
+		"shields" 200
+		"hull" 500
+		"mass" 22
+		"drag" .6
+		"heat dissipation" .92
+		"cargo space" 15
+		"outfit space" 56
+		"weapon capacity" 6
+		"engine capacity" 28
```

Comment:
This is a Syndicate drone, right? Maybe hold off on the release of the drone and then use X1050 Ion Engines (i.e. drop the engine space to 24). We could even change the lore of the X1050s to fit if neccessary, as the only reason they're released alongside the Barb is because of the Barb's low engine capacity; they could just as easily be said to be released for use on the Harvester, and that the Barb is just making use of past inventions.